src/app/indexedDb/wall-losses-surface-db.service.ts

```diff
@@ -15,6 +16,27 @@ export class WallLossesSurfaceDbService {
 
   }
 
+  insertDefaultMaterials(): Observable<number[]> {
```

Comment:
Should DefaultData.delete() be called?

any reason we are not also bringing in the id in the defaultMaterials?